### ISSUE TITLE
Various Accessibility Issues

### ISSUE THREAD

**Issue by stroomdev00** [gchq/stroom#3695]
I have a visually impaired user who is unable to develop translations due to various accessibility issues in v7.0 including fonts not scaling and high contrast making things disappear.  Whilst the UI in v7.2 is a complete rewrite, it would be useful to get this user's input to ensure that the later version is significantly better.

**Comment by stroomdev10** [comment]
Please demo 7.2 to the user for feedback

**Comment by stroomdev00** [comment]
Windows Settings: Theme = High Contrast Black, Display->Make Text bigger = max
Edge browser settings: Font size = very large
The above settings affect every application and shouldn’t be altered.
 
Stroom settings: Font Size = large
 
Data Preview – line spacing wrong although font size is ok.  Lines appear ‘crushed’ when using a large font.

Info – Looks fine.

Meta – same issues as Data Preview.
 
When selecting Choose Pipeline, popup is full height so not expecting resize area in lower right corner.  This area may even be below the edge of the screen so would never find it.  Resize (manual or auto) is needed to get a decent tree view of pipelines.  Alternatively, larger fonts require larger width popups.
 
Generally speaking, the monospaced fonts used within the ACE editor do not scale well.
 
Stepping – due to the theme, there’s no indication of which pipeline element has been highlighted.
 
The “Current Activity” below the browser tree pane needs to have a scroll bar and to be able to resize it to give more space for the browser tree.
 
The top right Menu should allow submenus to appear to the left rather than over the top of the main menu.  This is what happens with the top left menu (except submenus appear to the right).  Alternatively, get rid of the RH menu entirely – it’s a duplicate and only adds confusion to users.
 
Pipeline display within stepping – the arrows between elements are difficult to see as they’re grey and too small.
 
Green forward/back stepping buttons – in high contrast mode, green means ‘inactive’.  Would not even consider clicking one of these generally.  Should be changed to yellow (normal colour for hyperlink) or ideally white.  General comment for green anywhere too.
 
Dashboards – bottom scroll bar not visible until scrolled to the bottom, by which time you can longer see the column headings.  Also true in normal configuration.
 
 
Dashboard Query icons – would expect colours to disappear to match theme.
 
Dashboard queries – ability to write these in text form rather than GUI drag and drop would be beneficial, even for normal users.
 
Dashboard queries – when the disable button is used on a part of the query, that part is not greyed out due to the high contrast mode.  Maybe change to green for disabled?
 
Query, add term – when a new term is added, selecting it gives a dropdown list to select the field name.  If this dropdown list goes off the bottom of the screen, there’s no way to select later entries as the dropdown box appears to be fixed size.
 
When closing an unsaved tab, the ‘Confirm - xxx has unsaved changes…’ popup needs a scroll bar to be able to read the full message, although the at least the resize option does allow it to be dragged to a large size.
